plugins/modules/rds_cluster.py

```diff
@@ -189,12 +189,14 @@
         description:
           - Specifies the storage type to be associated with the DB cluster.
           - This setting is required to create a Multi-AZ DB cluster.
-          - When specified, a value for the I(iops) parameter is required.
+          - When specified I(storage_type=io1), a value for the I(iops) parameter is required.
+          - When I(engine) is set to aurora/aurora-mysql/aurora-postgresql, not specifying I(storage_type) defaults it to aurora standard.
```

Comment:
I would probably rephrase it
```suggestion
          - For Aurora DB clusters, I(storage_type) defaults to C(aurora) standard.
```

Reply:
Can you please ensure that the default behaviour is verified?